### PR TITLE
Y24-329 - Multiplexing documentation

### LINKED ISSUE #1396: Y24-239 - As developers we would like to add detailed documentation for pooling / multiplexing to inform other projects and users
**Describe the Housekeeping**

As developers we would like to add detailed documentation for  pooling / multiplexing to inform other projects and users:
- Create documentation in the traction repository (not confluence)
- Add any process diagrams as images
- Provide detailed documentation about how it works from a user perspective
- Provide detailed documentation about how it works technically
- Maybe create Architectural Decision Records (ADR)??

**Blocking issues**
Describe any other issues or tickets that may be blocking this change.

**Additional context**

### PR DESCRIPTION
**Pull request by BenTopping**
Closes #1396 

#### Changes proposed in this pull request

- Adds multiplexing overview containing the basic process.
- Adds pooling architecture diagrams and explanation.
- Adds user guide for traction-ui pooling interactions.
- Remove documentation from release tar ball.

Available at: https://sanger.github.io/traction-service/multiplexing/

### CODE REVIEW

File: documentation/multiplexing/mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+theme:
+  name: material
+  font: false
+  favicon: img/favicon.ico
+  palette:
+    # Palette toggle for automatic mode
+    - media: "(prefers-color-scheme)"
+      toggle:
+        primary: teal
+        accent: teal
```
Comment (dasunpubudumal) [inline documentation/multiplexing/mkdocs.yml:15]:
```suggestion
        primary: green
        accent: green
```

File: documentation/multiplexing/mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      accent: teal
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+
+    # Palette toggle for dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: slate
+      primary: teal
+      accent: teal
```
Comment (dasunpubudumal) [inline documentation/multiplexing/mkdocs.yml:32]:
```suggestion
      primary: green
      accent: green
```

File: documentation/multiplexing/mkdocs.yml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        primary: teal
+        accent: teal
+        icon: material/brightness-auto
+        name: Switch to light mode
+
+    # Palette toggle for light mode
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
+      primary: teal
+      accent: teal
```
Comment (dasunpubudumal) [inline documentation/multiplexing/mkdocs.yml:23]:
```suggestion
      primary: green
      accent: green
```

**Review by dasunpubudumal — APPROVED** [review]
Nice.

**Review by seenanair — APPROVED** [review]
Looking great, I got to learn few more things 👍 
Just a question, is 'User guide' better to be part of UI documentation?
Also, just passing on a review comment from Dasun on my PR on volume-tracking documentation as it is relevant to all
https://github.com/sanger/traction-ui/pull/1906#pullrequestreview-2275193626

**Comment by BenTopping** [comment]
> Looking great, I got to learn few more things 👍 Just a question, is 'User guide' better to be part of UI documentation? Also, just passing on a review comment from Dasun on my PR on volume-tracking documentation as it is relevant to all [sanger/traction-ui#1906 (review)](https://github.com/sanger/traction-ui/pull/1906#pullrequestreview-2275193626)

Ah ok ill bear the git diff changes in mind for the future 👍 

I spoke to @stevieing briefly about where the line between ui and service documentation is. I think it depends how we want to structure our docs. If we want step by step guides for each UI page it would make sense to move the user guide to the ui docs. But that also means users have to look in two places for the information and we currently can't link between UI and service docs easily.

**Comment by dasunpubudumal** [comment]
> > Looking great, I got to learn few more things 👍 Just a question, is 'User guide' better to be part of UI documentation? Also, just passing on a review comment from Dasun on my PR on volume-tracking documentation as it is relevant to all [sanger/traction-ui#1906 (review)](https://github.com/sanger/traction-ui/pull/1906#pullrequestreview-2275193626)
> 
> Ah ok ill bear the git diff changes in mind for the future 👍
> 
> I spoke to @stevieing briefly about where the line between ui and service documentation is. I think it depends how we want to structure our docs. If we want step by step guides for each UI page it would make sense to move the user guide to the ui docs. But that also means users have to look in two places for the information and we currently can't link between UI and service docs easily.

An index would be ideal I think.

**Pull request merged**